db/fixtures/miq_event_definitions.csv

```diff
@@ -70,6 +70,17 @@ request_host_shutdown,Host Shutdown Request,Default,host_operations
 request_host_reboot,Host Reboot Request,Default,host_operations
 request_host_enable_vmotion,Host Vmotion Enable Request,Default,host_operations
 request_host_disable_vmotion,Host Vmotion Disable Request,Default,host_operations
+
+#
+# Middleware Operation
+#
+hawkular_deployment,EAP Application Deployment,Default,middleware_operations
+hawkular_deployment_remove,EAP Application Undeployment,Default,middleware_operations
+hawkular_datasource,EAP Datasource Creation,Default,middleware_operations
+hawkular_datasource_remove,EAP Datasource Deletion,Default,middleware_operations
+hawkular_jdbc,EAP JDBC Driver Deployment,Default,middleware_operations
+hawkular_jdbc_remove,EAP JDBC Driver Undeployment,Default,middleware_operations
```

Comment:
Are you sure that we don't need the `.ok` suffixes in all of these? In the provider side these are defined `hawkular_jdbc_remove.ok` and so.

Also, I'd use "Middleware" instead of "EAP" to have, for example, `Middleware JDBC Driver Undeployment`

Reply:
The `.ok` and `.error` might be a legit concern. I'll investigate it tomorrow.

The "Middleware" and "EAP" is a part of requirements here, see [JMAN4-200](https://issues.jboss.org/browse/JMAN4-200). @jdoyleoss has to decide.

Reply:
@israel-hdez you were right, we need that. Do we want only successful events, or the failed ones as well, @jdoyleoss?

Like this:
```csv
hawkular_deployment.ok,EAP Application Deployment,Default,middleware_operations
hawkular_deployment.error,EAP Application Deployment Failed,Default,middleware_operations
hawkular_deployment_remove.ok,EAP Application Undeployment,Default,middleware_operations
hawkular_deployment_remove.error,EAP Application Undeployment Failed,Default,middleware_operations
hawkular_datasource.ok,EAP Datasource Creation,Default,middleware_operations
hawkular_datasource.error,EAP Datasource Creation Failed,Default,middleware_operations
hawkular_datasource_remove.ok,EAP Datasource Deletion,Default,middleware_operations
hawkular_datasource_remove.error,EAP Datasource Deletion Failed,Default,middleware_operations
hawkular_jdbc.ok,EAP JDBC Driver Deployment,Default,middleware_operations
hawkular_jdbc.error,EAP JDBC Driver Deployment Failed,Default,middleware_operations
hawkular_jdbc_remove.ok,EAP JDBC Driver Undeployment,Default,middleware_operations
hawkular_jdbc_remove.error,EAP JDBC Driver Undeployment Failed,Default,middleware_operations
```